core/src/client/client.rs

```diff
@@ -826,6 +826,15 @@ impl TermInfo for Client {
             .map(|state| state.metadata().unwrap().expect("Metadata always exist"))
             .map(|metadata| metadata.current_term_id())
     }
+
+    fn term_common_params(&self, id: BlockId) -> Option<CommonParams> {
```

Comment:
Should it return `Option`?
It seems `None` is returned only when we try to get the `CommonParams` of the genesis block, and it's practically not going to happen.

Reply:
`miner::prepare_block` and `block::enact` reads `term_common_params` with the parent header. The genesis block is an expected input for the function so it should return None.

Reply:
ACK. I think we can remove them though.
I'll try to change the return type in https://github.com/CodeChain-io/codechain/pull/1896 when refactoring them.